x-pack/plugins/dashboard_enhanced/scripts/storybook.js

```diff
@@ -9,5 +9,5 @@ import { join } from 'path';
 // eslint-disable-next-line
 require('@kbn/storybook').runStorybookCli({
   name: 'dashboard_enhanced',
-  storyGlobs: [join(__dirname, '..', 'public', 'components', '**', '*.story.tsx')],
```

Comment:
Would be nice to limit Storybook to just one place where all presentational components are. What do you think?

Reply:
let's discuss in different thread https://github.com/elastic/kibana/pull/63108#discussion_r410163752